src/catcher.ts

```diff
@@ -268,4 +269,15 @@ export default class Catcher {
 
     return this.stackParser.parse(error as Error);
   }
+
+  /**
+   * Return some details
+   */
+  private getAddons(): object {
+    const { width, height }  = window.screen;
```

Comment:
it's better to pass real width and height got from `window.innerWidth` and `window.innerHeight`